pkgs/development/python-modules/mean-average-precision/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  numpy,
+  pandas,
+}:
+
+buildPythonPackage rec {
+  pname = "mean-average-precision";
+  version = "2024.01.05.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "bes-dev";
+    repo = "mean_average_precision";
+    rev = "refs/tags/${version}";
+    hash = "sha256-qo160L+oJsHERVOV0qdiRIZPMjvSlUmMTrAzThfrQSs=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    numpy
+    pandas
+  ];
+
+  pythonImportsCheck = [
+    "mean_average_precision"
+  ];
+
```

Comment:
Are there no tests?

Reply:
Nope, no tests.

Reply:
In those cases, we usually explicitly add `doCheck = false;` and a comment that says that there are no tests.
Not a blocker here of course.

Reply:
Let's fix it while it's hot.